src/libecalc/core/models/compressor/train/base.py

```diff
@@ -101,17 +101,20 @@ def evaluate_rate_ps_pd(
         )
 
         if self.data_transfer_object.calculate_max_rate:
+            # only calculate max standard rate for time steps with valid input
+            valid_idx = np.where([failure_status is None for failure_status in input_failure_status], 1, 0)
```

Comment:
can we create a failure_status enum: `NO_FAILURE = None` to make sure that is consistent and use that as check?